### ISSUE TITLE
45` rotated rect + Rounded reactangle tooltip

### ISSUE THREAD

**Issue by azfullstack2258** [xuopled/react-simple-tooltip#4]
![screen shot 2017-12-11 at 03 04 34](https://user-images.githubusercontent.com/29067217/33817855-887f9df4-de20-11e7-8d45-b63939858d21.png)
When I set opacity of background, I could know how you made this tooltip shape.

**Comment by cedricdelpoux** [comment]
Hi @azfullstack2258, I understand your problem but I don't know how to handle it. For now, the only advice I can give you it to set arrow to 0 to keep the bubble only.

Maybe I can draw my arrow using a CSS triangle but if you set a border to your tooltip, there will be a separation between the bubble and the arrow

**Comment by properCase** [comment]
What about injecting an svg to be the arrow rather than a css block?

**Comment by cedricdelpoux** [comment]
Good idea @properCase. I'm open if you would like to contribute to this project 🙂
